ldes-server-admin/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/admin/domain/view/service/ViewServiceImpl.java

```diff
@@ -43,10 +49,25 @@ public void addView(ViewSpecification viewSpecification) {
 			throw new ExistingResourceException(VIEW_TYPE, viewSpecification.getName().asString());
 		}
 
+		checkViewForDuplicateRetentionPolicies(viewSpecification);
+
 		eventPublisher.publishEvent(new ViewAddedEvent(viewSpecification));
 		viewRepository.saveView(viewSpecification);
 	}
 
+	private void checkViewForDuplicateRetentionPolicies(ViewSpecification viewSpecification) {
+		viewSpecification.getRetentionConfigs().stream()
+				.map(retentionPolicy -> retentionPolicy.listStatements(null, RdfConstants.RDF_SYNTAX_TYPE, (Resource) null).nextStatement())
+				.collect(Collectors.groupingBy(Statement::getObject, Collectors.counting()))
+				.entrySet().stream()
+				.filter(entry -> entry.getValue() > 1)
+				.map(Map.Entry::getKey)
+				.findFirst()
```

Comment:
Can you include all duplicates in the exception at once? Having to adjust 1 thing and retrying and then adjusting 1 thing again is not very user friendly. It is better to include all issues at once.